hydromt_sfincs/sfincs.py

```diff
@@ -1011,6 +1016,126 @@ def setup_cn_infiltration(self, cn, antecedent_moisture="avg", reproj_method="me
         self.config.pop("qinf", None)
         self.set_config(f"{mname}file", f"sfincs.{mname}")
 
+    # Function to create curve number for SFINCS including recovery term (Kr)
+    def setup_curvenumber_infiltration_withrecovery(
+        self, data_catalog, name_files, reclass_table, effective, block_size=2000
+    ):
+        """Setup model the Soil Conservation Service (SCS) Curve Number (CN) files for SFINCS
+        including recovery term based on the soil saturation
+
+        Parameters
+        ---------
+        data_catalog    : data catalog used in the analysis
+        name_files      : list with name of the landcover, hydrologic soil group (HSG) and Saturated Hydraulic Conductivity (Ksat)
```

Comment:
I would prefer to split `name_files` into multiple arguments, called `lulc`, `soil_group` and `ksat`. I think that would be more clear to users. Also its possible to pass the data (xarray.DataArray) directly if you are working from python, so the argument does not have to be names, see e.g. `qinf` in `setup_constant_infiltration`.

Reply:
OK, just changed that.